koku/api/report/ocp/provider_map.py

```diff
@@ -208,6 +228,12 @@ def __init__(self, provider, report_type):
                             "cost_total": self.cloud_infrastructure_cost_by_project
                             + self.markup_cost_by_project
                             + self.cost_model_cost,
+                            "cost_distributed": self.cloud_infrastructure_cost_by_project
+                            + self.markup_cost_by_project
+                            + self.cost_model_cost
+                            + self.cost_model_distributed_cost_by_project,
+                            "cost_platform_distributed": self.platform_distributed_cost_by_project,
+                            "cost_worker_unallocated": self.worker_unallocated_distributed_cost_by_project,
```

Comment:
`cost_worker_unallocated_distributed` ? I know it's wordy, but the other costs have "distributed" and it is a distributed cost.